challenge.md

```diff
@@ -6,39 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git clone 'url'
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git diff
```

Comment:
Yep! We'd also probably want to use `git status` at around the same time to see which files have been created, moved, renamed, or destroyed.